src/Views/Form/Templates/Classic/resources/views/receipt.php

```diff
@@ -1,110 +1,125 @@
-		</p>
-		<?php require 'social-sharing.php'; ?>
+
+		<div class="receipt-sections">
+
 		<?php
-		/* @var Section $section */
-		foreach ( $receipt as $section ) {
-			// Continue if section does not have line items.
-			if ( ! $section->getLineItems() ) {
-				continue;
-			}
-
-			if ( 'PDFReceipt' === $section->id ) {
-				$pdfReceiptLinkDetailItem = $section['receiptLink'];
-				continue;
-			}
-
-			echo '<div class="details">';
-			if ( $section->label ) {
-				printf( '<h3 class="headline">%1$s</h3>', $section->label );
-			}
-			echo '<div class="details-table">';
-
-			/* @var LineItem $lineItem */
-			foreach ( $section as $lineItem ) {
-				// Continue if line item does not have value.
-				if ( ! $lineItem->value ) {
+			/* @var Section $section */
+			foreach ( $receipt as $section ) :
+				if ( ! $section->getLineItems() || 'PDFReceipt' === $section->id )
 					continue;
-				}
-
-				// This class is required to highlight total donation amount in receipt.
-				$detailRowClass = '';
-				if ( DonationReceipt::DONATIONSECTIONID === $section->id ) {
-					$detailRowClass = 'totalAmount' === $lineItem->id ? ' total' : '';
-				}
-
-				printf(
-					'<div class="details-row%1$s">%2$s<div class="detail">%3$s</div><div class="value">%4$s</div></div>',
-					$detailRowClass,
-					$lineItem->icon,
-					$lineItem->label,
-					$lineItem->value
-				);
-			}
-			echo '</div>';
-			echo '</div>';
-		}
-		?>
-
-		<?php if ( $pdfReceiptLinkDetailItem ) : ?>
-			<div class="give-btn download-btn">
-				<?php echo $pdfReceiptLinkDetailItem->value; ?>
+			?>
+
+			<div class="details">
+				<?php if ( $section->label ): ?>
+					<h3 class="headline">
+						<?= $section->label; ?>
+					</h3>
+				<?php endif; ?>
+				<div class="details-table">
+
+					<?php
+						/* @var LineItem $lineItem */
+						foreach ( $section as $lineItem ):
+							if ( ! $lineItem->value )
+								continue;
+
+							$class = '';
+							if ( DonationReceipt::DONATIONSECTIONID === $section->id ) {
+								$class = 'totalAmount' === $lineItem->id ? 'total' : '';
+							}
+						?>
+
+						<div class="details-row <?= $class; ?>">
+							<?= $lineItem->icon; ?>
+							<div class="detail"><?= $lineItem->label; ?></div>
+							<div class="value"><?= $lineItem->value; ?></div>
+						</div>
+					<?php endforeach; ?>
+
+				</div>
 			</div>
-		<?php endif; ?>
 
-	</div>
-	<div class="form-footer">
-		<div class="secure-notice">
-			<i class="fas fa-lock"></i>
-			<?php _e( 'Secure Donation', 'give' ); ?>
+		<?php endforeach; ?>
 		</div>
-	</div>
-</div>
 
+		<div class="dashboard-link-container">
+			<a class="dashboard-link" href="<?= esc_url( $donorDashboardUrl ); ?>">
+				<?= esc_html__( 'Go to my Donor Dashboard', 'give' ); ?>
+			</a>
+			<?php if ( isset( $section[ 'receiptLink' ] ) ) : ?>
+				<div class="give-btn download-btn">
+					<?= $section[ 'receiptLink' ]->value; ?>
+				</div>
+			<?php endif; ?>
+		</div>
+	</div>
```

Comment:
Closing article tag.

```suggestion
	</article>
```